### DISCUSSION TITLE
Many plots - does RAM of processor matter ?

### DISCUSSION THREAD

**Discussion by cashmeoussidexx** [Chia-Network/chia-blockchain#1770] [Q&A]
Hello, if I plan to have many plots on my server, does amount of RAM of processor matter ? To be specific, my server has 100 TB HDD and I want to fill it all with chia plots and I was wondering if I need some super powerful processor or big amount of RAM to run so many plots...

**Comment by depado** [comment]
Hello. No, there's no need for a super efficient processor or a huge amount of RAM to farm. The only step where you need some power is the plotting process, but running the full node and farming with your plots isn't CPU or RAM intensive. For example my full node is on a raspberry4 and it's working like a charm at 1.3 avg load and 8% RAM usage (4 cores/8GB RAM).
